addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -146,8 +147,26 @@ def _compute_plan_id(self):
                 scheduler.plan_id = False
 
     @api.depends('res_model_id', 'res_ids')
-    def _compute_plan_date_deadline(self):
-        self.plan_date_deadline = False
+    def _compute_plan_date(self):
+        self.plan_date = False
+
+    @api.depends('plan_date', 'plan_id')
+    def _compute_plan_summary(self):
+        for scheduler in self:
+            template_ids = scheduler.plan_id.template_ids
+            if not template_ids:
+                scheduler.plan_summary = False
+                continue
+            summaries = []
+            for template in template_ids:
+                summaries.append(Markup('<li>%s</li>') % (
+                    template.activity_type_id.name +
+                    (f": {template.summary}" if template.summary else '') +
+                    # We don't display deadlines when the user doesn't specify a plan_date
+                    (f" ({format_date(self.env, template._get_date_deadline(scheduler.plan_date))})"
+                     if scheduler.plan_date else '')
+                ))
+            scheduler.plan_summary = Markup('<ul>%s</ul>') % Markup().join(summaries)
```

Comment:
Maybe we could split the string formation (I find combination of "+" and if/else quickly hard to follow, but that's personal taste), like
```suggestion
                summary = template.activity_type_id.name
                if template.summary:
                    summary += f": {template.summary}"
                # We don't display deadlines when the user doesn't specify a plan_date
                if scheduler.plan_date
                    summary += f" {format_date(self.env, template._get_date_deadline(scheduler.plan_date))}"
                summaries.append(Markup('<li>%s</li>') % summary)
            scheduler.plan_summary = Markup('<ul>%s</ul>') % Markup().join(summaries)
```